media-gfx/photoqt/photoqt-4.7.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+ECM_KDEINSTALLDIRS=false
+_KFSLOT=6
```

Comment:
This is an eclass-internal variable, so wrong to be using here. If the goal was to make the build system take the `KF6/Qt6` route and have Qt6 (B)DEPENDs, set `KFMIN` to something `>=6` then.

Reply:
It was.